### PR TITLE
Umbraco Cloud Release notes December

### PR DESCRIPTION
**Pull request by sajumb**

### CODE REVIEW

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+## Key Takeaways
+
+* **Secrets Management** - Secure your secrets in Azure key vaults in an intuitive user interface and easy access to secrets during the run-time of your Umbraco solution.
+* **Organization Project view** - You will now see a lot more information for each of your organization projects on the project page under your organization.
+* **Updated public access page** - We have added the option to specify a name when you choose to specify a new entry to the allowlist for an environment.
+* **Kudu quicklink in Projects Dashboard** - You now have the option to navigate to Kudu for a cloud project environment from the "Projects Dashboard".
+
+## [Secrets Management](../set-up/project-settings/secrets-management.md)
+
+The new _secrets management_ feature in Umbraco Cloud enables you to store your secrets (digital authentication credentials such as private keys, database passwords, and 3rd-party API tokens) securely in Azure key vaults. Hereby, you will never have to worry about exposing the most sensitive information of your Umbraco Cloud solution in code repositories.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:12]:
```suggestion
The new _secrets management_ feature in Umbraco Cloud enables you to store your secrets securely in Azure key vaults.

This being digital authentication credentials such as private keys, database passwords, and 3rd-party API tokens

Hereby, you will never have to worry about exposing the most sensitive information of your Umbraco Cloud solution in code repositories.
```

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Storing Secrets Management in Azure key is enabled for all Umbraco cloud projects on Standard and Professional plans. 
+
+See the documentation section [Secrets Management](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/secrets-management) for more information.
+
+## Organization Project view
+
+As a member of a Umbraco Cloud organization, on the "Projects" page under your organization, you will find a more detailed overview of your projects than before.
+
+In addition to the name, plan, and status of the project, you can now also see the creation date, hosting region, payment status, and the number of environments running on the solution.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:27]:
```suggestion
In addition to the name, plan, and status of the project, you can now also see:
- Creation date.
- Hosting region.
- Payment status.
- The number of environments.
```

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+![ProjectsDashboardHoverEnvironment](images/OrganizationProjects.png)
+
+You can sort your organization projects ascending and descending for all columns in the projects table.
+
+## Public access page
+
+On the new version of the "Public Access" page, you can now specify a name when you add a new entry in the allowlist for an environment.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:35]:
```suggestion
On the "Public Access" page, you can now specify a name when you add a new entry in the allowlist for an environment.
```

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+
+![ProjectsDashboardHoverEnvironment](images/OrganizationProjects.png)
+
+You can sort your organization projects ascending and descending for all columns in the projects table.
+
+## Public access page
+
+On the "Public Access" page, you can now specify a name when you add a new entry in the allowlist for an environment.
+![PublicAccessV2](images/PublicAccessV2.gif)
+When you choose to add the public IP for the network you are currently working on, your name will automatically be used as the descriptive text.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:45]:
```suggestion
When choosing to add the public IP for the network you are currently working on, your name will automatically be used as the descriptive text.
```

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+
+## Kudu quicklink in Projects Dashboard
+
+And finally, we've added a quicklink to Kudo that you can see by hovering over an environment.
+![Kudu](images/Kudu.gif)
+You now have the opportunity to access both the cloud project's page, back office and Kudo from the "Projects Dashboard".
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:52]:
```suggestion
You now have the opportunity to access both the cloud project's page, backoffice and Kudo from the "Projects Dashboard".
```

File: umbraco-cloud/release-notes/2022-12-20-releasenotes.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+Every environment has its own key vault and the secured values can only be accessed by the associated environment. The secret will be referenced with the assigned name as an environment variable. During runtime of your Umbraco application, the secret can be accessed using standard .NET conventions such as
+_secretMessage = Environment.GetEnvironmentVariable("SecretName");
+
+Storing Secrets Management in Azure key is enabled for all Umbraco cloud projects on Standard and Professional plans. 
+
+See the documentation section [Secrets Management](https://docs.umbraco.com/umbraco-cloud/set-up/project-settings/secrets-management) for more information.
+
+## Organization Project view
+
+As a member of a Umbraco Cloud organization, on the "Projects" page under your organization, you will find a more detailed overview of your projects than before.
```
Comment (jonat123) [inline umbraco-cloud/release-notes/2022-12-20-releasenotes.md:29]:
```suggestion
As a member of a Umbraco Cloud organization, on the "Projects" page under your organization, you'll find a more detailed overview of your projects.
```

**Comment by jonat123** [comment]
@sajumb Looks good!

I've made some suggestions to shorten some of the sentences, so that it won't complain about vale :D 

It also complains about the use of easy, however I think it is okay to use it in this context 😄

**Comment by sajumb** [comment]
Great, thanks a lot. All suggestions have been merged.

**Comment by jonat123** [comment]
Great!

Merging it in :D

**Pull request merged**